README.md

```diff
@@ -26,3 +26,24 @@
   export class AppModule {
   }
   ```
+
+3. Add `RouterState` to main application state:
+
+  ```ts
+  import { RouterState } from '@ngrx/router-store';
+  
+  export interface AppState {
+    router: RouterState;
+  };
+  ```
+
+### Dispatching actions
+
+```ts
+import {routerActions} from '@ngrx/store-store';
+
+// ...
+store.dispatch(routerActions.go('/path', { query: 'string' ));
```

Comment:
I consider adding the `RouterState` interface to be optional. Thoughts @MikeRyan52 ?